pkg/vadmin/restart_node_vc.go

```diff
@@ -54,21 +54,17 @@ func (v *VClusterOps) RestartNode(ctx context.Context, opts ...restartnode.Optio
 func (v *VClusterOps) genStartNodeOptions(s *restartnode.Parms, certs *HTTPSCerts) *vops.VStartNodesOptions {
 	su := v.VDB.GetVerticaUser()
 	honorUserInput := true
-	opts := vops.VStartNodesOptions{
-		DatabaseOptions: vops.DatabaseOptions{
-			DBName:         &v.VDB.Spec.DBName,
-			RawHosts:       []string{s.InitiatorIP},
-			Ipv6:           vstruct.MakeNullableBool(net.IsIPv6(s.InitiatorIP)),
-			Key:            certs.Key,
-			Cert:           certs.Cert,
-			CaCert:         certs.CaCert,
-			UserName:       &su,
-			Password:       &v.Password,
-			HonorUserInput: &honorUserInput,
-		},
-		Nodes: s.RestartHosts,
-	}
-	// timeout option
+	opts := vops.VStartNodesOptionsFactory()
```

Comment:
Restructured code here so that we call the factor function for opts. This was needed to properly initialize a new field added to the opts.